models/calendar.yml

```diff
@@ -196,13 +196,16 @@
 
     - 9:00 PM:
         type: Talk
-        emoji: 😎
-        title: TBD
-        description: TBD
+        emoji: 🤖
+        title: JS Freaking Everywhere: A Chatbot Powered by AWS & NodeJS
```

Comment:
The colon after "Everywhere" is confusing the YAML parser. Would you please try surrounding the value in quotes? I think that should fix it.